### PR TITLE
Camera into neck

### PR DESCRIPTION
**Pull request by Nowp**

### CODE REVIEW

File: code/Modules/Items/PlayerInventory.cs [hunk 1]
```diff
@@ -64,5 +70,20 @@ public void Add(Item item, int position = 0)
...
+        private void RefreshViewModel()
+        {
+            if (null != HoldingItem)
+            {
+                ViewModel.SetHoldType(HoldingItem.HoldType);
+            }
+            else if (null == HoldingItem)
+            {
+                ViewModel.SetHoldType(HoldType.None);
+            }
```
Comment (Prakkkmak) [inline code/Modules/Items/PlayerInventory.cs:86]: HolfingItem == null ? blah : blouh
Better
ViewModel.SetHoldType(HoldingItem == nul ? blah : blouh)
Or you can set the hold type before

File: code/Player/HandViewModel.cs [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+        private Vector3 offset;
+
+        public HandViewModel()
+        {
+            SetModel(ViewModelPath);
+            SetBodyGroup(BodyGroupHead, 1);
+            SetBodyGroup(BodyGroupLegs, 1);
+            SetBodyGroup(BodyGroupFeet, 1);
+            SetAnimFloat(AnimKeyBodyWeight, 0f);
```
Comment (Prakkkmak) [inline code/Player/HandViewModel.cs:30]: Magic numbbbeeer

File: code/Player/MainPlayer.cs [hunk 3]
```diff
@@ -26,15 +25,27 @@ public MainPlayer()
...
+        public override void ClientSpawn()
+        {
+            base.ClientSpawn();
+
+            if (Inventory.ViewModel != null || !IsLocalPawn) return;
+            Inventory.ViewModel = new HandViewModel
+            {
+                EnableAllCollisions = false,
+                Owner = Owner,
+                EnableViewmodelRendering = true
```
Comment (Prakkkmak) [inline code/Player/MainPlayer.cs:44]: Owner after; isk its style

**Review by Prakkkmak — APPROVED** [review]

File: code/Player/MainPlayer.cs (hunk 3 above)
Comment (Nowp) [reply code/Player/MainPlayer.cs:44]:
```suggestion
                EnableAllCollisions = false,
                EnableViewmodelRendering = true,
                Owner = this.Owner
```

**Pull request merged**